inc/Plugin.php

```diff
@@ -241,10 +242,6 @@ private function init_common_subscribers() {
 			'critical_css_subscriber',
 			'sucuri_subscriber',
 			'expired_cache_purge_subscriber',
-			'preload_subscriber',
-			'sitemap_preload_subscriber',
-			'partial_preload_subscriber',
-			'fonts_preload_subscriber',
```

Comment:
This one should be preserved

Reply:
Look like theses doesn't exists anymore: ``			'sitemap_preload_subscriber',
			'partial_preload_subscriber',
			'fonts_preload_subscriber',``